tests/v2/validation/steve/vai/scripts/script.sh

```diff
@@ -14,9 +14,9 @@ check_go() {
 # Install Go if not already installed
 if ! check_go; then
     echo "Go not found. Installing Go..."
-    curl -L -o go1.22.4.linux-amd64.tar.gz https://go.dev/dl/go1.22.4.linux-amd64.tar.gz --insecure
-    tar -C /usr/local -xzf go1.22.4.linux-amd64.tar.gz
-    rm go1.22.4.linux-amd64.tar.gz
+    curl -L -o go1.23.1.linux-amd64.tar.gz https://go.dev/dl/go1.23.1.linux-amd64.tar.gz --insecure
+    tar -C /usr/local -xzf go1.23.1.linux-amd64.tar.gz
+    rm go1.23.1.linux-amd64.tar.gz
```

Comment:
Any reason for not using the same version as in go.mod? (eg: `1.23.4`)? I can see that before it was also using an older version (1.22.4 vs  1.22.7)

Reply:
No reason, just missed it. Thanks for catching it!